src/common/libflux/attr.h

```diff
@@ -66,6 +66,13 @@ int flux_get_size (flux_t *h, uint32_t *size);
  */
 const char *flux_get_hostbyrank (flux_t *h, uint32_t rank);
 
+/* Look up the broker.starttime attribute on rank 0.
+ * Subtract from flux_reactor_now() get Flux instance uptime.
+ * N.B. if the instance has been restarted, this value is the most
+ * recent restart time.
+ */
```

Comment:
this comment doesn't seem to match the code, we're not subtracting from `flux_reactor_now()`?

Reply:
oh wait, I think you mean, "Subtract the starttime from flux_reactor_now() to get the Flux instance uptime", not that the function does this.  Maybe minor reformat / wording, or maybe it's just me being stupid.

Reply:
Oh yeah I see now that the wording is a bit confusing.  I'll fix.